include/flatbuffers/grpc.h

```diff
@@ -22,49 +22,198 @@
 #include "grpc++/support/byte_buffer.h"
 #include "grpc/byte_buffer_reader.h"
 
+namespace flatbuffers {
+namespace grpc {
+
+enum AddRef { ADD_REF };
+enum StealRef { STEAL_REF };
+
+template <class T>
+class Message {
+ public:
+  inline Message()
+      : slice_(grpc_empty_slice()) {
```

Comment:
Do you know someone who you could pull in to the review this? I'd like to make sure it's correct, and I don't trust my "skim the grpc code for an afternoon" level of understanding.